components/nimbus/Cargo.toml

```diff
@@ -34,6 +34,7 @@ sha2 = "0.9"
 hex = "0.4"
 once_cell = "1"
 uniffi = { version = "^0.14", optional = true }
+chrono = { version = "0.4", features = ["serde"]}
```

Comment:
Added `chrono` as a dependency for convenience, checked its license, it's MIT/Apatche 2.0 licensed so we should be good. I'll make sure to regen the dependency summaries.

Its types already implement `serde::Serialize` and `serde::Deserialize` and its types can be built from strings, and `SystemTime` which allows me to keep the change here small.